lib/std/zig/AstGen.zig

```diff
@@ -13981,6 +13981,37 @@ fn lowerAstErrors(astgen: *AstGen) !void {
     var notes: std.ArrayListUnmanaged(u32) = .empty;
     defer notes.deinit(gpa);
 
+    const token_starts = tree.tokens.items(.start);
+    const token_tags = tree.tokens.items(.tag);
+    const parse_err = tree.errors[0];
+    const tok = parse_err.token + @intFromBool(parse_err.token_is_prev);
+    const tok_start = token_starts[tok];
+    const start_char = tree.source[tok_start];
+
+    if (token_tags[tok] == .invalid and (start_char == '\"' or start_char == '\'' or start_char == '\\' or start_char == '/')) {
```

Comment:
```suggestion
    if (token_tags[tok] == .invalid and (start_char == '\"' or start_char == '\'' or start_char == '/' or mem.startsWith(u8, "\\\\", tree.source[tok_start..]))) {
```
This avoids adding a misleading error for a single `\`.